fast64_internal/f3d/f3d_writer.py

```diff
@@ -407,13 +407,7 @@ def addCullCommand(obj, fMesh, transformMatrix, matWriteMethod):
     # if the object has a specifically set culling bounds, use that instead
     for vertexPos in obj.get("culling_bounds", obj.bound_box):
         fMesh.cullVertexList.vertices.append(
-            F3DVert(
-                Vector(vertexPos),
-                [0, 0],
-                Vector([0, 0, 0]),
-                None,
-                0.0,
-            ).toVtx(
+            F3DVert(Vector(vertexPos), [0, 0], Vector([0, 0, 0]), None, 0.0,).toVtx(
```

Comment:
Hey this might be the thing causing the black test to fail

Reply:
absolutely is. just run `black .` to format everything, it really should Just Work :tm: , if it doesn't we want to figure out why. Formatting isn't meant to be a pain

Reply:
I did and it didn't format this. Not sure why

Reply:
Can you run `black . --verbose` and share the output